styles/edit_mode.h

```diff
@@ -49,4 +55,8 @@ template<int DEFAULT_VALUE = 300> using IgnitionTime = Scale<IsLessThan<IntArg<I
 // RetractionTime = RETRACTION_TIME_ARG - uses "Auto" feature with WavLen when = 0
 template<int DEFAULT_VALUE = 0> using RetractionTime = Scale<IsLessThan<IntArg<RETRACTION_TIME_ARG,DEFAULT_VALUE>,Int<1>>,IntArg<RETRACTION_TIME_ARG,DEFAULT_VALUE>,WavLen<EFFECT_RETRACTION>>;
 
+// Bend Options for Ignition/Retraction effects - defaults to BEND = 32768
```

Comment:
Do we really need these aliases?

Reply:
They're meant to simplify the style code from the library where these are applied, not required but makes the code a little simpler when the feature is added.  If they're an issue I can remove.

Reply:
Aliases should be used sparingly, as they add complication in the style editor.
Also, they can hide what's actually going on sometimes, making it harder for the end user.
My first impression is that it would be better to not have these aliases.

Reply:
OK, I've removed.